src/classes/BGE_BatchGiftEntry_UTIL_TEST.cls

```diff
@@ -1,10 +1,46 @@
+/*
+    Copyright (c) 2019, Salesforce.org
+    All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
 /**
- * Created by lmeerkatz on 12/27/18.
- */
+* @author Salesforce.org
+* @date 2019
+* @group BGE
+* @group-content ../../ApexDocContent/BatchDataImport.htm
+* @description Unit Tests for the BGE Utility class.
+*/
 
 @isTest
 private with sharing class BGE_BatchGiftEntry_UTIL_TEST {
 
+    /*********************************************************************************************************
+    * @description Tests truncateList()
```

Comment:
Can we make the descriptions in this class more substantive?

Reply:
Can you give an example of what you think would be helpful? I'm not opposed, just not terribly creative...

Reply:
Yeah, just something other providing the name of the test in the description, otherwise I would argue we don't even need the description and can save some real estate!

Like for this one: "Tests truncating a list to various sizes, including negative numbers and numbers greater than the size of the list." (the latter two would also be good tests to add!)